frontend/src/views/dashboard/MyProblems.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { useHistory } from 'react-router-dom';
+import { MainHeaderText, SecondaryHeaderText } from '../../components/core/Text';
+import ProblemCard from '../../components/card/ProblemCard';
+import { TextLink } from '../../components/core/Link';
+import { useAppSelector } from '../../util/Hook';
+import {
+  CenteredContainer,
+  FlexHorizontalContainer,
+  FlexLeft,
+  RelativeContainer
+} from '../../components/core/Container';
+import { GreenSmallButton, TextButton } from '../../components/core/Button';
+import { TextInput } from '../../components/core/Input';
+import { Problem } from '../../api/Problem';
+
+type MyProblemsProps = {
+  loading: boolean,
+};
+
+const Content = styled.div`
+  text-align: left;
+`;
+
+const TopText = styled.div`
```

Comment:
```suggestion
const TopTextContainer = styled.div`
```
nit (personal preference): Thought this was text at first, this would just clear it up a tad.

Reply:
Sg - will implement this locally (since the component needs to be refactored as well)